daprdocs/content/en/concepts/dapr-services.md

```diff
@@ -0,0 +1,45 @@
+---
+type: docs
+title: "Dapr Services"
+linkTitle: "Services"
+weight: 100
+---
+
+
+Dapr supports multiple services that help sidecar - sidecar, sidecar - app and sidecar - system communication. Following are the services and the way to leverage them with your application.
+
+#### Daprd
+
+Dapr sidecar is available as Kubernetes container or as a Daprd process while using Dapr locally. Dapr runs Daprd by default when `Dapr Run` is executed through Dapr cli. While that's the default behavior, some users might still have to use Daprd explicity in certain special scenarios. For example a user may want to run sidecar in production system without running in a container. In cases like that when daprd can be executed through command line as follows
```

Comment:
Lets try to clarify this - `daprd` is the executable of the sidecar. It's good to put things in context (the CLI runs it as part of "dapr run", the sidecar injector injects into a pod when deployed to k8s etc.) but we need to make sure it is clear.

Also, I think it would be better to elaborate more when it would makes sense to run this and not use the CLI (the given scenario example is not accurate)